src/lib.rs

```diff
@@ -462,12 +462,13 @@ pub enum ProcError {
 ///
 /// If you compile with the optional `backtrace` feature (disabled by default),
 /// you can gain access to a stack trace of where the error happened.
-#[cfg_attr(feature = "serde1", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "serde1", derive(Serialize))]
```

Comment:
doesn't matter much, but i guess that instead of completely removing Deserialize you can limit it to something like:
```
#[cfg_attr(feature = "serde1", derive(Serialize))]
#[cfg_attr(all(feature = "serde1", not(feature = "backtrace")), derive(Deserialize))]
```

Reply:
Yeah, that could work, but I think I'm going to opt for the simplier implementation (just removing `Deserialize`) until someone has a concrete usecase where they need it